functions/source/cfct_register/cfct_register.py

```diff
@@ -0,0 +1,247 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
```

Comment:
Why does it have its own license info? Any reason we don't want to have Apache 2.0 license, same as repo license.

Reply:
we can use Apache 2.0 license at repo level, taking this out now.

Reply:
license updated